### DISCUSSION TITLE
Argon hitsound skinning?

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#24239] [Ideas] [answered]
tl;dr: would be nice to be able to skin the hitsounds in argon.

With the new o!taiko argon hitsounds out, I found the flourish sound a bit too cheesy for my liking. But it seems there's no way to change the hitsounds for the argon skin. Exporting the skin, and just dropping a new `taiko-strong-hitflourish.wav` in the folder didn't do anything, neither did putting it in `Argon/taiko-strong-hitflourish.wav`...
It would be nice if it was possible to change them, in some way.

If anything, this could technically be considered a lack of feature-parity with stable, as stable has the "user" skin, which is just overlaying a few changes onto the default skin, which is more or less what im doing here.

**Comment by peppy** [comment, marked as answer]
Could you try `Gameplay/Argon/taiko-strong-hitflourish.wav`?

**Reply by 2minwia** [reply]
Doesn't work :(

**Reply by peppy** [reply]
Interesting. It's not something we've tested so I'll have to double-check as to why.

**Reply by Susko3** [reply]
Adding folders to the zip and putting the file as `Gameplay/Argon/taiko-strong-hitflourish.wav` seems to work just fine here.

Test skin with only that sample added: [test.osk.zip](https://github.com/ppy/osu/files/15181085/test.osk.zip)
(Skin changes the sample to `shutter.wav` -- the F12 screenshot sound)

**Reply by SolsticeSpectrum** [reply]
Where do you put that? To exported argon skin?

**Reply by SolsticeSpectrum** [reply]
Also is there a particular place in this repository where I can find out what's all the stuff I can edit? Like is there a corresponding argon resources folder here?

**Reply by SolsticeSpectrum** [reply]
Either I'm doing something wrong or I don't know cuz your test.osk does not work for me

**Reply by Susko3** [reply]
> Where do you put that? To exported argon skin?

Should be obvious from the test skin I provided.

> Also is there a particular place in this repository where I can find out what's all the stuff I can edit? Like is there a corresponding argon resources folder here?

https://github.com/ppy/osu-resources/tree/master/osu.Game.Resources/Samples/Gameplay/Argon
But don't bet on this structure and filenames staying as-is.

> Either I'm doing something wrong or I don't know cuz your test.osk does not work for me

Turn up your effects, and tone down your music. The very first big blue hitobject on https://osu.ppy.sh/beatmapsets/2152060#taiko/4534320 has the screenshot (F12) sound for me with the test skin.

**Reply by SolsticeSpectrum** [reply]
It doesn't for me. Here is also my osk which also completely doesn't work, do you see any particular problem in my skin? Does it work for you? If it does then there must be something else wrong completely.
[-.avenir.sounds.-.rabidpan.osk.zip](https://github.com/ppy/osu/files/15193683/-.avenir.sounds.-.rabidpan.osk.zip)

**Reply by CsKLevente** [reply]
The test.osk.zip, from @Susko3, worked for me for the Taiko hitsound change. However putting the hitsounds for normal mode in the same folder didn't change the respective hitsounds.

Any idea where/how can we change the hitsounds for the default mode while keeping the rest of the Argon skin intact?

**Reply by Susko3** [reply]
For the osu! ruleset, you'll have to put it in the `Gameplay/` folder, eg. `Gamplay/normal-sliderslide.wav`.

**Comment by Luc1Carter** [comment]
why didn't it work out(

**Comment by JoogsWasTaken** [comment]
Mind me asking how you were able to export the skin? For me the option to export the default option is greyed out, so I'm unsure how I would go about replacing the flourish sound file (if it worked, which it apparently doesn't?)

![grafik](https://github.com/ppy/osu/assets/22560425/3e55efad-a31c-43bd-9c55-b637d1258d39)

I'm also not a fan of the flourishes personally. I like the new hitsounds overall, but the flourishes don't feel good to play with in most maps I've played. I'd rather mappers explicitly select notes for flourishes where they make sense.

**Reply by nagi-desuuu** [reply]
Opening the default skin in the layout editor creates a copy for the user to work with, and that duplicate can then be exported just like any other skin.
